src/lib/journey/callbacks/confirmation/confirmation.story.svelte

```diff
@@ -3,7 +3,7 @@
   import Centered from '$components/primitives/box/centered.svelte';
 
   import Confirmation from './confirmation.svelte';
-  import type { CallbackMetadata, StepMetadata } from '$journey/journey.interfaces';
+  import type { CallbackMetadata, StepMetadata } from '../..//journey.interfaces';
```

Comment:
Not sure why the build didn't catch this, but it looks like you have a double slash here.